.github/workflows/main.yml

```diff
@@ -0,0 +1,37 @@
+name: Build
+
+on: [push, pull_request]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-go@v1
+      with:
+        go-version: 1.13.x
+    - run: |
+        export PATH="$PATH:$(go env GOPATH)/bin"
+        make lint
+  test:
+    needs: lint
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-18.04', 'macOS-10.14', 'windows-2019']
+        go: ['1.13.x']
+        cgo: ['0', '1']
+    name: Go ${{ matrix.go }} on ${{ matrix.os }} CGO_ENABLED=${{ matrix.cgo }}
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go }}
+    - run: |
+        make test
+        cd cmd/osstat && go build && ./osstat
+      shell: bash
+      env:
+        CGO_ENABLED: ${{ matrix.cgo }}
+        GO111MODULE: on
```

Comment:
I believe we don't need GO111MODULE specification anymore, since `auto` in Go 1.13 activates module-aware mode when the directory contains go.mod (Ref: https://golang.org/doc/go1.13#modules)

Is my understanding wrong?

Reply:
That's right. I will delete this.